extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -409,11 +409,11 @@ export class AmpA4A extends AMP.BaseElement {
                         if (isValid) {
                           return creativeParts.creative;
                         }
-                        return Promise.reject(
-                            'Key failed to validate creative\'s signature.');
+                        throw new Error("Key failed to validate creative's " +
```

Comment:
Let's discuss that in [another issue](https://github.com/ampproject/amphtml/issues/5889). For now, this keeps the same behavior.